app/components/DocumentTitle/__tests__/DocumentTitle.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+
+import DocumentTitle from '..';
+
+describe('DocumentTitle', () => {
+  afterEach(cleanup);
```

Comment:
afterEach(cleanup) can be removed if there is only test.

Reply:
Done (https://github.com/Amsterdam/registraties/pull/107/commits/66193688f624ffd138fef74db0d54969d46e0aca)